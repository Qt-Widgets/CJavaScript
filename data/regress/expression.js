console.log(1+2);
console.log(1-2);
console.log(1*2);
console.log(1/2);
console.log(1%2);

console.log(-5);

console.log(3<8);
console.log(3>8);
console.log(3<=8);
console.log(3>=8);
console.log(3==8);
console.log(3!=8);

console.log(1+"a");
