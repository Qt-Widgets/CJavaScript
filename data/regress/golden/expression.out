3
-1
2
0.5
1
-5
true
false
true
false
false
true
1a
